pkg/k8s/portforward.go

```diff
@@ -172,7 +176,12 @@ func (pf *PortForward) run() error {
 		return err
 	}
 
-	return fw.ForwardPorts()
+	err = fw.ForwardPorts()
+	if err != nil {
+		err = fmt.Errorf("%s for %s/%s", err.Error(), pf.namespace, pf.podName)
```

Comment:
You can just have `err` without the `.Error()` part

Reply:
Thanks for pointing this. I improved this :smiley: